### ISSUE TITLE
Bug(?): Muzzleflashes are dimmer than they should be

### ISSUE THREAD

**Issue by Kestrellius** [scp-fs2open/fs2open.github.com#3356]
When muzzleflashes appear in-game, they are often darker and more transparent than their animations suggest they should be. Even pure FFFFFF white will not be completely opaque. This problem becomes more or less severe under different conditions. It's not entirely clear what the conditions are, but I think it might have something to do with the muzzleflash clipping into the ship's geometry.

[A test case, with pure white bitmaps.](https://youtu.be/AUiBrd5EPXk)

Note that it's possible to see where the flashes are overlapping with each other and the laser bolts, despite the fact that they should all be pure opaque white. Also, contrast the lasers -- which are pure white, functioning correctly -- with the dimmer muzzleflashes.

[Another example, from FOTG, with actual art.](https://youtu.be/h_-BLyNVVgE)

Again, this muzzleflash bitmap is, in its central regions, pure FFFFFF white. It should be totally opaque at the center.

Observe the way the flashes become brighter or dimmer as the angle of viewing changes.

The inability to reliably create bright muzzleflashes is a problem, and I would like it if they were rendered as bright as their bitmaps.

**Comment by Kestrellius** [comment]
The animations used in the test cases:
[muzzletesteffects1.zip](https://github.com/scp-fs2open/fs2open.github.com/files/6251704/muzzletesteffects1.zip)
[muzzletesteffects2.zip](https://github.com/scp-fs2open/fs2open.github.com/files/6251705/muzzletesteffects2.zip)

**Comment by Baezon** [comment]
Gonna un-assign myself from this. As mentioned in #3479, beyond what I've done, I'm out of my depth.

**Comment by wookieejedi** [comment]
#3479 fixed the issue for all intents and purposes, muzzle flashes look nice and bright now.

**Comment by Kestrellius** [comment]
Yeah, I'm in agreement. This has more or less fully remedied the concerns I had. Though as Asteroth has said, I guess it's still probably worth looking into the remaining minor dimness for "why is this happening and will it cause any other problems" reasons.

**Comment by MjnMixael** [comment]
Closing based on the conversation above.
